grails-app/controllers/au/org/ala/biocollect/BioActivityController.groovy

```diff
@@ -1925,6 +1925,12 @@ class BioActivityController {
                             in = ParameterIn.PATH,
                             required = true,
                             description = "Project id"
+                    ),
+                    @Parameter(
+                        name = "force",
+                        in = ParameterIn.QUERY,
+                        description = "a flag to control how to retrieve archive file",
```

Comment:
replace with "Set to true to generate Darwin Core Archive on demand (slow) or false to get the pre-generated file (might not have the latest data)"